src/db/db.ts

```diff
@@ -1,5 +1,22 @@
 import CouchDB from "../lib/CouchDB";
 
-const db = new CouchDB();
+// get defaults from environment
+const scheme = import.meta.env.DB_SSL === "true" ? "https" : "http";
+const domain = import.meta.env.DB_DOMAIN;
+const port = import.meta.env.DB_PORT;
+const dbName = import.meta.env.DB_NAME;
+const user = import.meta.env.DB_USER;
+const pass = import.meta.env.DB_PASS;
+const logLevelEnv: string = import.meta.env.LOGLEVEL?.toUpperCase() || "DEBUG";
```

Comment:
Moving these environment variables in the the db module.  This allows us to keep the CouchDB and Logger Libraries more modular and not dependent on environment variables.